gossipsub_test.go

```diff
@@ -1923,13 +1925,11 @@ func TestGossipSubLeaveTopic(t *testing.T) {
 	connect(t, h[0], h[1])
 
 	// Join all peers
-	var subs []*Subscription
 	for _, ps := range psubs {
-		sub, err := ps.Subscribe("test")
```

Comment:
Wait, this changes the semantics of the test...

Reply:
How so? We never use `subs`.

Reply:
uhm ok yhen, i thought they were used.